src/kagglehub/gcs_upload.py

```diff
@@ -66,16 +69,27 @@ def _upload_blob(file_path: str, model_type: str):
         token_exception = "'token' field missing from response"
         raise BackendError(token_exception)
 
+    # Define chunk size (e.g., 1 MB)
+    chunk_size = 1024 * 1024
+    headers = {"Content-Type": "application/octet-stream"}
+
     with open(file_path, "rb") as f:
```

Comment:
Where do you "seek" on the file to ensure you start uploading where you left off?